### DISCUSSION TITLE
content-docs-theme deprecated ?

### DISCUSSION THREAD

**Discussion by apptut** [nuxt/content#1361] [Q&A]
I can not find the doc or url  about content-docs-theme any more? It already deprecated? Is there has a new for replacement?

**Comment by nozomuikuta** [comment]
@apptut 

Content Docs Theme is deprecated as announced in #726.

Authors announced that the successor, Docus, will be public at some point.
You can check what it looks like in [the current Nuxt Content documentation](https://content.nuxtjs.org/).
